src/components/OnchainBounties/Components/FilterByTags.tsx

```
@@ -0,0 +1,143 @@
+	}, [bounties]);
+
+	useEffect(() => {
+		const filtered = availableTags.filter((tag) => tag.toLowerCase().includes(searchInput.toLowerCase()));
+		setFilteredTags(filtered);
+	}, [searchInput, availableTags]);
+
+	const handleSetTags = (tag: string) => {
+		if (tag && !tags.includes(tag)) {
+			const newTags = [...tags, tag];
+			setTags(newTags);
+			setSelectedTags && setSelectedTags(newTags);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)